File: common/src.old/main/yummyloop/yummy/old/mixin/item/Elytra.java
package yummyloop.yummy.old.mixin.item;

import net.minecraft.item.DyeableItem;
import net.minecraft.item.ElytraItem;
import org.spongepowered.asm.mixin.Mixin;

@Mixin(ElytraItem.class)
public final class Elytra implements DyeableItem {
}

File: src/main/yummyloop/example/mixin/item/Elytra.kt
package yummyloop.example.mixin.item

import net.minecraft.item.DyeableItem
import net.minecraft.item.ElytraItem
import org.spongepowered.asm.mixin.Mixin

@Mixin(ElytraItem::class)
class Elytra : DyeableItem{
}
